doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -533,39 +533,41 @@ The soil moisture deficit :math:`D_{irrig}` is
 
    D_{irrig} = \left\{
    \begin{array}{lr}
-   w_{thresh} - w_{avail} &\qquad w_{thresh} > w_{avail} \\
+   w_{target} - w_{avail} &\qquad w_{thresh} > w_{avail} \\
    0 &\qquad w_{thresh} \le w_{avail}
    \end{array} \right\}
 
-where :math:`w_{thresh}` is the irrigation moisture threshold (mm) and :math:`w_{avail}` is the available moisture (mm). The moisture threshold is
+where :math:`w_{target}` is the irrigation target soil moisture (mm)
```

Comment:
- [ ] Could you add back the definition of $w_{avail}$ here? It doesn't seem to be defined anywhere else.

Reply:
It is defined just below in equation 2.26.23, least in my html build.